.travis.yml

```diff
@@ -1,9 +1,26 @@
+branches:
+  only:
+    - master
```

Comment:
I'd leave it on for all branches - it's useful for PR's

Reply:
it will still run for all PRs and tags, just not for other branches on this repo. if you would push a branch to the repo right now and open a PR for it then CI would run twice, once for the branch and once for the PR. this PR is preventing this double run.

Reply:
Understood